frontend/cypress/integration/common/istio_config.ts

```diff
@@ -301,6 +301,17 @@ Then('user only sees {string}', (sees: string) => {
   });
 });
 
+Then('user only sees items related to {string}', (sees: string) => {
+  let count:number;
+  cy.request('GET', "/api/istio/config?objects=gateways&validate=true").should((response) =>{
```

Comment:
the same goes here, should not be hardcoded "gateways"

Reply:
the problem is that Istio generates Apps and Gateways for k8s gateways, which is causing conflicts.